go.mod

```diff
@@ -190,7 +190,6 @@ require (
 replace google.golang.org/grpc => google.golang.org/grpc v1.63.2
 
 replace (
-	github.com/go-jose/go-jose/v4 => github.com/go-jose/go-jose/v4 v4.0.5 // CVE-2025-27144
 	// controller runtime
```

Comment:
We can just bump in the indirect call. 
It has no need for replaces

Reply:
That was the point... but in this case there is no reference to it anywhere, so we are removing it.